main.py

```diff
@@ -192,6 +192,10 @@ def on_handshake_completed_request_flow_list(self, event):
         if switch.is_enabled():
             self._request_flow_list(switch)
 
+    def handle_handshake_completed_request_flow_list(self, switch):
+        """Request an flow list right after the handshake is completed."""
+        self._request_flow_list(switch)
+
```

Comment:
I believe this should be part of `on_handshake_completed_request_flow_list`, shouldn't it?

Reply:
You're right @italovalcy, I extracted this method but then forgot to call it where it was originally supposed to be, done here https://github.com/kytos-ng/of_core/pull/60/commits/902877f2068897dda54bb130a14318ab87452ff2.